src/agents/model_settings.py

```diff
@@ -104,11 +104,11 @@ class ModelSettings:
     metadata: dict[str, str] | None = None
     """Metadata to include with the model response call."""
 
-    store: bool | None = None
+    store: bool | None = True
     """Whether to store the generated model response for later retrieval.
     Defaults to True if not provided."""
```

Comment:
indeed, this needs to be accurate; this should be:
- for Responses API, when you don't pass true here, storing data is automatically enabled
- for Chat Completions API, when you don't pass true here, storing dat is disabled